cdap-ui/app/cdap/components/FastAction/LogAction/index.js

```diff
@@ -27,35 +27,39 @@ export default class LogAction extends Component {
 
     this.state = {
       isDisabled: false,
-      runId: null,
-      startTime: null,
-      endTime: null,
+      runId: props.entity.runId,
+      startTime: props.entity.startTime,
+      endTime: props.entity.endTime,
```

Comment:
I don't think the startTime and endTime is actually being used anywhere. Might be able to get away without it?? I think it was my left over code :(